### PR TITLE
Skill learn very base implement

### PR DESCRIPTION
**Pull request by S-anasol**
Cant handle tabs correctly in VS(default settings)
So few lines changed tabulation :(

Added:
* skills.txt - database with some dummy skills
* db table with learned skills
* loading/saving skill list for character in DB.
* skill learning in-game.
* modified ZC_ABILITY_LIST to read "real" skills data, but i have no research experience to get it work with client correctly(properly display learned skills in client).

P.S.
Still not a C# dev. 
С# experience is a few hours of copypasting :+1:

### CODE REVIEW

File: src/ChannelServer/Network/ChannelPacketHandler.cs [hunk 1]
```diff
@@ -806,49 +806,72 @@ public void CZ_CLIENT_HIT_LIST(ChannelConnection conn, Packet packet)
 		public void CZ_REQ_NORMAL_TX_NUMARG(ChannelConnection conn, Packet packet)
 		{
 			var size = packet.GetShort();
-			var type = packet.GetShort(); // 1 - statpints, 2 - skillpoints
-			var count = packet.GetInt();
-			//Throws exception when trying distributing skill points (count 6)
-			if (count != 5)
-				throw new Exception("Unknown CZ_REQ_NORMAL_TX_NUMARG format, expected 5 stats.");
```
Comment (epicmiro) [inline src/ChannelServer/Network/ChannelPacketHandler.cs]: Not sure if you should throw new exceptions vs debug logging. Haven't looked trough aura source lately to confirm though.. @exectails ?

File: src/Shared/Data/Database/Skills.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Aura development team - Licensed under GNU GPL
+// For more information, see license file in the main folder
```
Comment (epicmiro) [inline src/Shared/Data/Database/Skills.cs]: Skill and Skills cause confusion. Does melia also use SkillManager like Aura or is there no such system present?

**Comment by epicmiro** [comment]
Use tabs instead of spaces, Aura projects follow this indentation rule.

File: src/Shared/Data/Database/Skills.cs (hunk 2 above)
Comment (S-anasol) [inline src/Shared/Data/Database/Skills.cs]: There are no skills system, @exectails didn't code it yet :)

**Pull request closed without merging**